extensions/s3/src/main/java/io/deephaven/extensions/s3/S3SeekableChannelProviderPlugin.java

```diff
@@ -29,9 +29,12 @@ public SeekableChannelsProvider createProvider(@NotNull final URI uri, @Nullable
         if (!isCompatible(uri, config)) {
             throw new IllegalArgumentException("Arguments not compatible, provided uri " + uri);
         }
-        if (!(config instanceof S3Instructions)) {
+        if (config != null && !(config instanceof S3Instructions)) {
```

Comment:
Is this the correct error message? Suggestion: "Only S3Instructions are valid when reading files from S3"